hw2/03-count.html

```diff
@@ -6,28 +6,42 @@
     <title>03 - Count</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
-    <main>
-      <h1>Exercise 03 - Count</h1>
-      <hr />
-      <div>
-        <label>Highlight all occurences of a word:</label>
-        <input />
-      </div>
-      <hr />
-      <div>
-        It was the best of times, it was the worst of times, it was the age of
-        wisdom, it was the age of foolishness, it was the epoch of belief, it
-        was the epoch of incredulity, it was the season of Light, it was the
-        season of Darkness, it was the spring of hope, it was the winter of
-        despair, we had everything before us, we had nothing before us, we were
-        all going direct to Heaven, we were all going direct the other way—in
-        short, the period was so far like the present period, that some of its
-        noisiest authorities insisted on its being received, for good or for
-        evil, in the superlative degree of comparison only.
+  <body style="background-color: #00425c">
```

Comment:
Small optional suggestion. I remember in intro to web dev reading about css best practices and I think it recommended to use css style sheets rather than inline style attributes if you can. nothing wrong with using attributes but just a memory this line triggered. Also seems like its only one line so might be all good.

Reply:
Thanks so much for your feedback, Christian! It's always in-depth and helpful.  I considered using a stylesheet, but since It was only line, for this exercise I made the decision to use it inline.  In the future, I will separate out my custom css in its own file, as I agree that is best practice. :)